<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 239, 244);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QFrame" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>111</width>
      <height>720</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 64);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="homeBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>81</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(46, 52, 64);
	color: rgb(236, 239, 244);
}

QPushButton:hover {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>81</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(46, 52, 64);
	color: rgb(236, 239, 244);
}

QPushButton:hover {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}</string>
     </property>
     <property name="text">
      <string>Sort</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adminBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>111</width>
       <height>81</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(46, 52, 64);
	color: rgb(236, 239, 244);
}

QPushButton:hover {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(236, 239, 244);
	color: rgb(46, 52, 64);
}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="contentContainer" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>1181</width>
      <height>701</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1181</width>
      <height>701</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1181</width>
      <height>701</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 239, 244);</string>
    </property>
    <widget class="QWidget" name="homeContent" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>1181</width>
       <height>701</height>
      </rect>
     </property>
     <widget class="QPushButton" name="infoBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>660</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(46, 52, 64);
	color: rgb(236, 239, 244);
}
</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
     <widget class="QPushButton" name="contactBtn">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>660</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border:none;
	font: 75 12pt &quot;Consolas&quot;;
	background-color: rgb(46, 52, 64);
	color: rgb(236, 239, 244);
}
</string>
      </property>
      <property name="text">
       <string>Contact</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sortContent" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>701</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>1161</width>
        <height>611</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adminContent" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>1181</width>
       <height>701</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="adminPlaceholder">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Admin Placeholder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>adminContent</zorder>
    <zorder>sortContent</zorder>
    <zorder>homeContent</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>homeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayHome()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displaySort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adminBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayAdmin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contactBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayContact()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>987</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>1276</x>
     <y>692</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infoBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>688</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayAdmin()</slot>
  <slot>displayHome()</slot>
  <slot>displaySort()</slot>
  <slot>displayContact()</slot>
  <slot>displayHelp()</slot>
 </slots>
</ui>
